sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/snowflake--package.txt

```diff
@@ -101,7 +101,7 @@ LogicalTableScan(table=[[myicebergtable_1]])
         },
         {
           "name" : "Query.MyIcebergTable-8",
-          "sql" : "CREATE OR REPLACE VIEW Query.MyIcebergTable-8(id, customer_id, loan_type_id, amount, duration, application_date, updated_at, __pk)\nAS SELECT id, customer_id, loan_type_id, amount, duration, application_date, updated_at, 1 AS __pk\nFROM myicebergtable_1\nWHERE id = ? AND updated_at = ?\nORDER BY updated_at DESC NULLS LAST, 1"
+          "sql" : "CREATE OR REPLACE VIEW Query.MyIcebergTable-8(id, customer_id, loan_type_id, amount, duration, application_date, updated_at, __pk)\nAS SELECT id, customer_id, loan_type_id, amount, duration, application_date, updated_at, 1 AS __pk\nFROM myicebergtable_1\nWHERE id = ? AND updated_at = ?\nORDER BY updated_at DESC NULLS LAST, '1'"
```

Comment:
What's that '1' there at the end. That seems redundant. Not a new issue. Just noticed it.

Reply:
this looks like an issue, it should ordinal 1 not a char literal

Reply:
I had to fix it for postgres, ill see if theres a more generic way. I'm not sure why calcite is now doing this. https://github.com/apache/calcite/blob/a6a1e2cef332893fd90286098869c56529e052c3/core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java#L610

Reply:
Actually, it should not be an ordinal and we've been introducing an extra incorrect sort (?). In this query we have a '1' as the primary key, but we've been adding it as an ordinal in postgres

Reply:
This an issue in postgres as it doesn't support sorting by char literals

Reply:
We add a 1 literal as the primary key when a table doesn't have a primary key (e.g. when you aggregate globally without a group by). Not sure why this is happening here. Let me take a look.